src/components/NavBar/OidcLoginComponent.jsx

```diff
@@ -1,6 +1,6 @@
 // React Imports
 import React, { useState } from 'react';
-// Inrupt Library Imports
+// Custom Hook Imports
 import { useSession } from '@hooks';
 // Material UI Imports
 import { TextField, Box, Button } from '@mui/material';
```

Comment:
We've been importing MUI individually for performance - which we haven't actually tested how much of a performance hit importing it this way is, but the docs say the other way is better for performance.

Reply:
Got this changed to individual imports.